src/presentational/Home.jsx

```diff
@@ -10,10 +10,19 @@ export default class Home extends Component {
   render() {
     return (
       <Layout>
-        <div className="home-buttons container-fluid">
-          <h1>I am a...</h1>
-          <Link className='btn btn-info' role='button' to='/teachers'>Teacher</Link>
-          <Link className='btn btn-info' role='button' to='/students'>Student</Link>
+        <div className='home'>
+          <div className='jumbotron'>
+           <div className='page-header'>
+            <h1>Welcome to Wildcam Gorongosa Education Portal</h1>
```

Comment:
Two comments here,
* Wording: seems a wee bit odd; recommend either "Welcome to **the** Wildcam Gorongosa Education Portal" _or_ "Welcome to Wildcam **Gorongosa's** Education Portal"
* (Minor) Code formatting: did the tab indent size go from 2 to 1 in this section?